src/ray/object_manager/plasma/store.cc

```diff
@@ -1033,4 +1062,21 @@ void PlasmaStore::DoAccept() {
                                               boost::asio::placeholders::error));
 }
 
+void PlasmaStore::ProcessCreateRequests(size_t num_bytes_space) {
+  for (auto request = create_request_queue_.begin();
+      request != create_request_queue_.end(); ) {
+    RAY_LOG(DEBUG) << "Reprocessing queued create request";
+    auto status = HandleCreateObjectRequest(request->first, request->second);
+    if (status.IsTransientObjectStoreFull()) {
+      // The object store is still full.
+      // NOTE(swang): There could be other requests behind this one that are
+      // actually serviceable. This may be inefficient, but eventually this
+      // request will get served and unblock the following requests, once
+      // enough objects have been spilled.
+      break;
+    }
+    request = create_request_queue_.erase(request);
```

Comment:
Does it move iterator to the next one automatically? (i.e., don't we need request++ here?)

Reply:
Yes! https://en.cppreference.com/w/cpp/container/list